src/app/(school)/(reviews)/professor/[slug]/loading.tsx

```diff
@@ -0,0 +1,14 @@
+import { ReviewItemSkeleton } from "@/common/components/ReviewItem/ReviewItemSkeleton";
+import { ReviewSection } from "@/common/components/ReviewSection";
+
+export default function Loading() {
+  return (
+    <ReviewSection>
+      <ReviewItemSkeleton />
+      <ReviewItemSkeleton />
+      <ReviewItemSkeleton />
+      <ReviewItemSkeleton />
+      <ReviewItemSkeleton />
```

Comment:
Question: is there a better way to handle the number of skeleton being rendered? Not sure if this will be a concern but there might be cases where there is only 1 review but we will render 5 skeletons still

Reply:
imo: i don't think this would be too big of an issue, the skeleton doesnt necessarily need to reflect exactly as how the UI will eventually render

it is also impractical to query db to check how many reviews there are in order to accurately render x numbers of reviews since the additional db query is unjustifiable for just a slightly more accurate display of a loading placeholder

Reply:
Thanks for the clarification! It is indeed true that it will be impractical to query db for this. I think ftm we can stick with the current implementation until we see a need to change it in the future